src/journal-remote/journal-remote-write.c

```diff
@@ -75,6 +92,9 @@ int writer_write(Writer *w,
                 r = do_rotate(&w->journal, w->mmap, file_flags);
                 if (r < 0)
                         return r;
+                r = journal_directory_vacuum(w->output, w->metrics.max_use, w->metrics.n_max_files, 0, NULL, true);
```

Comment:
For new code, please always comment boolean arguments with their name:
`/* verbose = */ true`

Reply:
done